hack/reference-docs-template/members.tpl

```diff
@@ -0,0 +1,48 @@
+{{ define "members" }}
```

Comment:
Once we have this in pipelines as well, it would be nice to have this in plumbing and vendor it from there instead of having to do this is each Tekton sub project.

Reply:
Agreed!